modules/custom/az_person/config/install/core.entity_view_display.node.az_person.az_card.yml

```diff
@@ -103,10 +103,25 @@ third_party_settings:
         attributes: ''
         effect: none
         speed: fast
+    group_title_link:
+      children:
+        - smart_title
+      label: 'Title link'
+      parent_name: group_card_body
+      region: content
+      weight: 12
+      format_type: link
+      format_settings:
+        classes: ''
+        show_empty_fields: false
+        id: ''
+        target: custom_uri
+        custom_uri: '[node:az-canonical-url]'
+        target_attribute: default
```

Comment:
My suggestion from my comment was not quite right, as it causes the heading style to override the link color:

<img width="846" alt="image" src="https://user-images.githubusercontent.com/74572157/217895152-af51133f-088c-4b13-bcc8-af97fdda543b.png">

To fix this and match the original HTML, I added a "Title Heading" field group around the link. Hopefully this suggestion works:

```suggestion
    group_title_heading:
      children:
        - group_title_link
      label: 'Title Heading'
      parent_name: group_card_body
      region: content
      weight: 1
      format_type: html_element
      format_settings:
        classes: 'h5 mt-0'
        show_empty_fields: false
        id: ''
        element: h3
        show_label: false
        label_element: h3
        label_element_classes: ''
        attributes: ''
        effect: none
        speed: fast
    group_title_link:
      children:
        - smart_title
      label: 'Title Link'
      parent_name: group_title_heading
      region: content
      weight: 1
      format_type: link
      format_settings:
        classes: ''
        show_empty_fields: false
        id: ''
        target: custom_uri
        custom_uri: '[node:az-canonical-url]'
        target_attribute: default
```